homeassistant/components/vesync/fan.py

```diff
@@ -46,6 +48,7 @@
     "EverestAir": [FAN_MODE_AUTO, FAN_MODE_SLEEP, FAN_MODE_TURBO],
     "Vital200S": [FAN_MODE_AUTO, FAN_MODE_SLEEP, FAN_MODE_PET],
     "Vital100S": [FAN_MODE_AUTO, FAN_MODE_SLEEP, FAN_MODE_PET],
+    "SmartTowerFan": [FAN_MODE_ADVANCED_SLEEP, FAN_MODE_AUTO, FAN_MODE_TURBO],
```

Comment:
The modes defined on [SmartTowerFan](https://github.com/webdjoe/pyvesync/blob/632be01b9c8e03854c3ea1f98921f1ea26ee8626/src/pyvesync/vesyncfan.py#L151) are `['normal', 'auto', 'advancedSleep', 'turbo', 'off'],`. Do we not need "auto" ?

Reply:
Auto is already in the list.  I have now added normal though.

Reply:
* Shouldn't FAN_MODE_NORMAL be "normal" and not "turbo" ?
* Also it seems that we should be invoking the method [normal_mode](https://github.com/webdjoe/pyvesync/blob/632be01b9c8e03854c3ea1f98921f1ea26ee8626/src/pyvesync/vesyncfan.py#L1906C9-L1906C20) for that mode?

Reply:
Corrected - I shouldn't have tried to fix it while getting out the door. :)